ruff_lsp/server.py

```diff
@@ -588,46 +595,76 @@ def apply_organize_imports(arguments: tuple[TextDocument]):
     uri = arguments[0]["uri"]
     text_document = LSP_SERVER.workspace.get_document(uri)
     LSP_SERVER.apply_edit(
-        _create_workspace_edits(
-            text_document, _formatting_helper(text_document, only="I001") or []
-        ),
+        _create_workspace_edits(text_document, _fix_helper(text_document, only="I001")),
         "Ruff: Format imports",
     )
 
 
-def _formatting_helper(
-    document: workspace.Document, *, only: str | None = None
+if os.environ.get("RUFF_BETA_INTERNAL"):
+
+    @LSP_SERVER.command("ruff.applyFormat")
+    def apply_format(ls: server.LanguageServer, arguments: tuple[TextDocument]):
+        apply_format_impl(ls, arguments)
```

Comment:
A bundler could be useful here so that we could have two different builds. But I don't see how we could achieve the same with python. I guess what we really want are feature flags and your `RUFF_BETA_INTERNAL` is kind of that

Reply:
Python has extras but they unfortunately don't persist to runtime